ui/lib/ldap/addon/components/page/configure.hbs

```diff
@@ -78,14 +78,11 @@
 </form>
 
 {{#if this.showRotatePrompt}}
-  <Modal
-    @title="Rotate your root password?"
-    @type="info"
-    @isActive={{this.showRotatePrompt}}
-    @showCloseButton={{true}}
-    @onClose={{fn (mut this.showRotatePrompt) false}}
-  >
-    <section class="modal-card-body">
+  <Hds::Modal id="ldap-rotate-password-modal" @onClose={{fn (mut this.showRotatePrompt) false}} as |M|>
```

Comment:
Just out of curiousity - do modals always need id's now?

Reply:
Not that I can tell - but I wondered if screen readers used them. I also thought that we could use them in a general test selector `.hds-modal#${id}`

Reply:
Thanks for clarifying, Claire! I think it's a great pattern to add the ids to each modal :)